### ISSUE TITLE
Make application functional on touch screen devices

### ISSUE THREAD

**Issue by adamjarling** [adamjarling/loc-speculative-annotations#60]
Make application functional on touch screen (specifically text and stamps tool) (ig Ipad/chromebook, mobile is not a priority).

Note this is dependent upon Stamp tool refactor issue

**Comment by adamjarling** [comment]
@CourtneyMcClellan @jessvu I'm cleaning up the handling of some extra touch events which were firing on mobile/tablet, and interfering with the Type and Stamp tool functionality on mobile.  Now this should be working better.

I do have a question on handling zoom in/out for touch.   With lots of objects added to the app, it feels clunky to use pinch zoom in/out for touch.  Should I try and disable zooming via pinch zoom on touch, and rely on the same zoom buttons we're using on desktop?

**Comment by jessvu** [comment]
@adamjarling this is a good consideration. I am going to have to take some time to think through this one, but I can get back to you by the end of the week.

**Comment by CourtneyMcClellan** [comment]
@jessvu @adamjarling Do to time constraints we are going to put this on the backburner.

**Comment by adamjarling** [comment]
Sounds good.  @CourtneyMcClellan I've tried to keep tablet/mobile display somewhat in line with @jessvu 's Figma layout designs, so I'd consider the app "functional" on mobile, but perhaps not as refined as the desktop experience.

Regarding the zoom in/out for mobile, implementing the same approach as desktop (disabling pinch zoom and just letting the zoom control buttons handle zooming) would be a fairly straightforward change (not too much time).  It'd give added benefit of making the app UX (zoom functions at least) better in mobile, though it might not be intuitive to how users expect to interact with touch devices.    

To sum up, disabling pinch zoom and just using the zoom buttons may provide a better baseline functionality for mobile, and not much cost in terms of time.

**Comment by CourtneyMcClellan** [comment]
If there are low effort way to improve mobile then I say do it. As you
mentioned before Adam, I don't want mobile to be off putting. I think
overall, we are looking at our short timeline.

**Comment by adamjarling** [comment]
After playing with this, I'm not sure disabling pinch zoom is any better.  Seems about the same trade-off with and without it.

**Comment by adamjarling** [comment]
I'm going to close this for now.  We could open a new, more focused issue if need be.
